### DISCUSSION TITLE
How to properly change naming output template? (or possible error)

### DISCUSSION THREAD

**Discussion by startingstuffup** [JunkFood02/Seal#1521] [Q&A] [answered]
I've currently set it to:
`%title.200B [%(id)s] - %(uploader)s.%(ext)s`
So it should save the name as Title - Uploaded By but it's not downloading the uploader and I think I broke the app since it's no longer downloading anything (I'll just reinstall though). I'm not sure if it's a yt-dlp error or in Seal. Or maybe I'm doing it wrong? I don't want to keep constantly renaming things manually whenever I download stuff

Edit: here's the one I'm now using that's working as I'd wanted:
%(title).200B - %(uploader)s.%(ext)s

**Comment by JunkFood02** [comment, marked as answer]
`%(title).200B` you forgot the brackets

**Reply by startingstuffup** [reply]
This is embarrassing. I actually haven't forgotten the brackets but I was downloading a file that I'd just recently deleted, and for some reason, Seal thought the file still existed. But with a fresh install and your suggestion, it's working perfectly now. Thanks!
